data_structures/k_tree/test_k_tree.py

```diff
@@ -27,3 +27,29 @@ def test_valid_breadth(basic_tree):
     basic_tree.insert(2, basic_tree.root.val)
     basic_tree.breadth_first_traversal(bread.append)
     assert bread[1].val > bread[0].val
+
+
+def test_basic_valid_print_level(basic_tree):
+    assert basic_tree.print_level_order() == '''1
+'''
+
+
+def test_valid_print_level(basic_tree):
+    basic_tree.insert(2, basic_tree.root.val)
+    assert basic_tree.print_level_order() == '''1
+2
+'''
+
+
+def test_complex_valid_print_level(basic_tree):
+    basic_tree.insert(2, basic_tree.root.val)
+    basic_tree.insert(3, basic_tree.root.val)
+    basic_tree.insert(4, 2)
+    basic_tree.insert(5, 2)
+    basic_tree.insert(6, 3)
+    basic_tree.insert(7, 6)
+    assert basic_tree.print_level_order() == '''1
+23
+456
+7
+'''
```

Comment:
If you want to do this on one line, you can use the new-line character to the same effect.
```python
assert basic_tree.print_level_order() == '1\n23\n456\n7\n'
```